### ISSUE TITLE
Texte d'une autre collection mélangé 

### ISSUE THREAD

**Issue by mingarao** [ccahier/weboai#9]
Un texte de la collection Molière apparait dans la collection Montesquieu : 
http://weboai.cahier.huma-num.fr/set/montesquieu

@glorieux-f

**Comment by glorieux-f** [comment]
Intéressant. Serait-ce un essai de copié-collé pour avoir un modèle ?

**Comment by mingarao** [comment]
Non, car pas de mention de ce texte dans le sitemap correspondant :  

http://montesquieu:montesquieu@montesquieu.huma-num.fr/rest/bvm/bvm-sitemap.xml

**Comment by glorieux-f** [comment]
Tu as trouvé la raison ? Si tu relances une indexation, cela revient à chaque fois ?

**Comment by mingarao** [comment]
En relançant une indexation, le problème a disparu ;)

Tu as fait des modifs ? Je ne vois pas de push sur le git.

Merci en tout cas pour ton suivi :+1: :+1:

**Comment by glorieux-f** [comment]
L'indexation a dû se faire sur un état de fichier ou l'éditeur avait laissé un lien modèle ou quelque chose comme ça.
Je te laisse le plaisir de fermer l'issue.
